### PR TITLE
Global substitutions using {...}.

### PR DESCRIPTION
**Pull request by samscott89**

### CODE REVIEW

**Comment by dhatch** [comment]
I understand for Java, but what's the point of having this for Python & Ruby?

**Comment by dhatch** [comment]
Besides the gemfile example.

**Comment by samscott89** [comment]
> I understand for Java, but what's the point of having this for Python & Ruby?

I've been hit before a bunch by assuming pip install will update, but it wont unless you do `pip install --upgrade oso`. Giving them the full version makes it more likely they wont hit this.

File: docs/getting-started/quickstart.rst [hunk 1]
```diff
@@ -57,39 +57,37 @@ Adding oso
...
+        $ gem install oso-oso -v {release}
 
     .. group-tab:: Java
 
-      oso supports Java versions **>= 9**
+      oso v{release} supports Java versions **>= 9**
 
-      Download :download:`oso-0.2.5.jar </examples/quickstart/java/lib/oso-0.2.5.jar>`.
+      Go to the `Maven Repository <https://github.com/osohq/oso/packages/321403>`_
+      and download the latest jar
```
Comment (gj) [inline docs/getting-started/quickstart.rst:82]:
```suggestion
      and download the latest jar.
```

**Pull request merged**